Packs/GitLab/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "GitLab",
     "description": "Pack for handling gitlab operations",
     "support": "xsoar",
-    "currentVersion": "1.0.0",
```

Comment:
Is this is a new Pack in marketplace_v2?
If yes - Leave it at 1.0.0 and we'll force-merge it.

Reply:
OK